pytest-extension/exasol/pytest_extension/__init__.py

```diff
@@ -105,10 +66,129 @@ def func(conn_name: str, path_in_bucket: str | None = None) -> None:
         else:
             bucketfs_params = backend_aware_bucketfs_params
         if backend == BACKEND_ONPREM:
-            _create_bucketfs_connection_onprem(pyexasol_connection, conn_name, bucketfs_params)
+            create_bucketfs_conn_object_onprem(pyexasol_connection, conn_name, bucketfs_params)
         elif backend == BACKEND_SAAS:
-            _create_bucketfs_connection_saas(pyexasol_connection, conn_name, bucketfs_params)
+            create_bucketfs_conn_object_saas(pyexasol_connection, conn_name, bucketfs_params)
         else:
             raise ValueError(f'Unsupported backend {backend}')
 
     return func
+
+
+@pytest.fixture(scope="session")
+def onprem_database_std_params(use_onprem,
```

Comment:
Std? What should mean Here?

Reply:
Standard database parameter. That's how I call them in PEC.

Reply:
I didn't have the background, because I didn't see the change in PEC

Reply:
I was lucky, being forced to face the PEC in advance :slightly_smiling_face: 
But also thanks for the nice introduction to @ahsimb